includes/Core/Util/Reset_Persistent.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Reset_Persistent
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class providing functions to reset the persistent plugin settings.
+ *
+ * @access private
+ * @ignore
+ */
```

Comment:
```suggestion
/**
 * Class providing functions to reset the persistent plugin settings.
 *
 * @since n.e.x.t
 * @access private
 * @ignore
 */
```